### ISSUE TITLE
Crash when loading server with Dungeons Mobs v4.0.4, Forge 1.19.2--43.2.0

### ISSUE THREAD

**Issue by villainous-j** [Thelnfamous1/Dungeons-Mobs#173]
### Issue
Since updating to Dungeons Mobs v4.0.4, my Forge 43.2.0 server isn't able to start. Downgrading (only Dungeons Mobs) to v4.0.3 starts the server correctly.

### Failure message
```
Dungeons Mobs (dungeons_mobs) has failed to load correctly
java.lang.NoClassDefFoundError: net/minecraft/client/player/Input
```

### Crash report
[crash-2023-02-08_01.25.42-fml.txt](https://github.com/Infamous-Misadventures/Dungeons-Mobs/files/10683908/crash-2023-02-08_01.25.42-fml.txt)

**Comment by PixlJinx** [comment]
Having the same issue but I'm on Forge 43.2.4

**Comment by villainous-j** [comment]
@PixlJinx Glad to ensure the issue isn't version specific! Any chance you could also share your crash report? I'm looking for any clues as to what might be going on.

**Comment by DeltaWolf7** [comment]
Same for me rolled back to 4.0.3 until fixed, will try pull logs this aft.

**Comment by DeltaWolf7** [comment]
`[modloading-worker-0/ERROR] [ne.mi.fm.ja.FMLModContainer/LOADING]: Failed to register automatic subscribers. ModID: dungeons_mobs, class com.infamous.dungeons_mobs.DungeonsMobs
[09.02 12:21:59] [Server] java.lang.NoClassDefFoundErrornet/minecraft/client/player/Input
[09.02 12:21:59] [Server] 	at java.lang.Class.getDeclaredConstructors0(Native Method) ~[?:?] {re:mixin}
[09.02 12:21:59] [Server] 	at java.lang.Class.privateGetDeclaredConstructors(Class.java:3373) ~[?:?] {re:mixin}
[09.02 12:21:59] [Server] 	at java.lang.Class.getConstructor0(Class.java:3578) ~[?:?] {re:mixin}
[09.02 12:21:59] [Server] 	at java.lang.Class.getConstructor(Class.java:2271) ~[?:?] {re:mixin}
[09.02 12:21:59] [Server] 	at net.minecraftforge.eventbus.api.EventListenerHelper.computeListenerList(EventListenerHelper.java:75) ~[eventbus-6.0.3.jar%2351!/:?] {}
[09.02 12:21:59] [Server] 	at net.minecraftforge.eventbus.api.EventListenerHelper.lambda$getListenerListInternal$0(EventListenerHelper.java:56) ~[eventbus-6.0.3.jar%2351!/:?] {}
[09.02 12:21:59] [Server] 	at net.minecraftforge.eventbus.LockHelper.computeIfAbsent(LockHelper.java:61) ~[eventbus-6.0.3.jar%2351!/:?] {}
[09.02 12:21:59] [Server] 	at net.minecraftforge.eventbus.LockHelper.get(LockHelper.java:44) ~[eventbus-6.0.3.jar%2351!/:?] {}
[09.02 12:21:59] [Server] 	at net.minecraftforge.eventbus.LockHelper.computeIfAbsent(LockHelper.java:39) ~[eventbus-6.0.3.jar%2351!/:?] {}
[09.02 12:21:59] [Server] 	at net.minecraftforge.eventbus.api.EventListenerHelper.getListenerListInternal(EventListenerHelper.java:56) ~[eventbus-6.0.3.jar%2351!/:?] {}
[09.02 12:21:59] [Server] 	at net.minecraftforge.eventbus.api.EventListenerHelper.getListenerList(EventListenerHelper.java:50) ~[eventbus-6.0.3.jar%2351!/:?] {}
[09.02 12:21:59] [Server] 	at net.minecraftforge.eventbus.EventBus.addToListeners(EventBus.java:276) ~[eventbus-6.0.3.jar%2351!/:?] {}
[09.02 12:21:59] [Server] 	at net.minecraftforge.eventbus.EventBus.register(EventBus.java:269) ~[eventbus-6.0.3.jar%2351!/:?] {}
[09.02 12:21:59] [Server] 	at net.minecraftforge.eventbus.EventBus.registerListener(EventBus.java:163) ~[eventbus-6.0.3.jar%2351!/:?] {}
[09.02 12:21:59] [Server] 	at net.minecraftforge.eventbus.EventBus.lambda$registerClass$2(EventBus.java:86) ~[eventbus-6.0.3.jar%2351!/:?] {}
[09.02 12:21:59] [Server] 	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183) ~[?:?] {}
[09.02 12:21:59] [Server] 	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179) ~[?:?] {}
[09.02 12:21:59] [Server] 	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179) ~[?:?] {}
[09.02 12:21:59] [Server] 	at java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:992) ~[?:?] {}
[09.02 12:21:59] [Server] 	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509) ~[?:?] {}
[09.02 12:21:59] [Server] 	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499) ~[?:?] {}
[09.02 12:21:59] [Server] 	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150) ~[?:?] {}
[09.02 12:21:59] [Server] 	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173) ~[?:?] {}
[09.02 12:21:59] [Server] 	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?] {}
[09.02 12:21:59] [Server] 	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596) ~[?:?] {}
[09.02 12:21:59] [Server] 	at net.minecraftforge.eventbus.EventBus.registerClass(EventBus.java:86) ~[eventbus-6.0.3.jar%2351!/:?] {}
[09.02 12:21:59] [Server] 	at net.minecraftforge.eventbus.EventBus.register(EventBus.java:126) ~[eventbus-6.0.3.jar%2351!/:?] {}
[09.02 12:21:59] [Server] 	at net.minecraftforge.fml.javafmlmod.AutomaticEventSubscriber.lambda$inject$6(AutomaticEventSubscriber.java:61) ~[javafmllanguage-1.19.2-43.2.4.jar%23941!/:?] {}
[09.02 12:21:59] [Server] 	at java.util.ArrayList.forEach(ArrayList.java:1511) ~[?:?] {re:mixin}
[09.02 12:21:59] [Server] 	at net.minecraftforge.fml.javafmlmod.AutomaticEventSubscriber.inject(AutomaticEventSubscriber.java:48) ~[javafmllanguage-1.19.2-43.2.4.jar%23941!/:?] {}
[09.02 12:21:59] [Server] 	at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:78) ~[javafmllanguage-1.19.2-43.2.4.jar%23941!/:?] {}
[09.02 12:21:59] [Server] 	at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$10(ModContainer.java:121) ~[fmlcore-1.19.2-43.2.4.jar%23940!/:?] {}
[09.02 12:21:59] [Server] 	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) [?:?] {}
[09.02 12:21:59] [Server] 	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796) [?:?] {}
[09.02 12:21:59] [Server] 	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) [?:?] {}
[09.02 12:21:59] [Server] 	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) [?:?] {}
[09.02 12:21:59] [Server] 	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) [?:?] {re:mixin,re:computing_frames}
[09.02 12:21:59] [Server] 	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) [?:?] {re:mixin,re:computing_frames}
[09.02 12:21:59] [Server] 	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) [?:?] {re:mixin}
[09.02 12:21:59] [Server] Caused byjava.lang.ClassNotFoundException: net.minecraft.client.player.Input
[09.02 12:21:59] [Server] 	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:141) ~[securejarhandler-2.1.4.jar:?] {}
[09.02 12:21:59] [Server] 	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?] {}
[09.02 12:21:59] [Server] 	... more`

**Comment by villainous-j** [comment]
Looks to be fixed with v4.0.6! Closing this down.
